sixMans/sixMans.py

```diff
@@ -221,6 +221,29 @@ async def q(self, ctx):
         if six_mans_queue._queue_full():
             await self._randomize_teams(ctx, six_mans_queue)
 
+    @commands.guild_only()
+    @commands.command(aliases=["forcequeue"])
+    @checks.admin_or_permissions(manage_guild=True)
+    async def fq(self, ctx, *, member: discord.Member):
+        """Adds a specific member to the queue [Testing only]"""
+        await self._pre_load_queues(ctx)
+        await self._pre_load_games(ctx, False)
+        six_mans_queue = self._get_queue(ctx)
+        player = member
+
+        if player in six_mans_queue.queue.queue:
+            await ctx.send(":x: You are already in the {0} queue".format(six_mans_queue.name))
+            return
+        for game in self.games:
+            if player in game:
+                await ctx.send(":x: You are already in a game")
+                return
+
+        automove = await self._get_automove(ctx)
```

Comment:
This variable doesn't look like it's used here at all and we end up getting it again in the randomize teams method so I assume this is just left over from testing

Reply:
This was leftover code from when I was deciding where the behavior should exist. It is unused and should be removed.

Reply:
Done